lib/commands/config/index.js

```diff
@@ -119,7 +121,7 @@ class ConfigCommand extends Command {
                     type: 'input',
                     name: 'dbname',
                     message: 'Enter your Ghost database name:',
-                    default: this.instance.config.get('database.connection.database', `ghost_${this.system.environment}`)
+                    default: this.instance.config.get('database.connection.database', `ghost_${this.system.environment}_${path.resolve(process.cwd()).substr(path.resolve(process.cwd()).lastIndexOf('/') + 1)}`)
```

Comment:
@vikaspotluri123 possibly - although at this point none of the validation has been run on any of the arguments. We'd technically have access to the argv.name command line arg but the default service name (if none is provided) is based off of the URL - which wouldn't necessarily have been defined at this point.